pages/common/neofetch.md

```diff
@@ -0,0 +1,23 @@
+# neofetch
+
+> CLI system tool to display information about your operating system, software and hardware.
+
+- Call neofetch to return the default config, and create it if it's the first time the program runs:
+
+`neofetch`
+
+- Pass specific parameters to alter what gets displayed:
+
+`neofetch --{{opt1}} {{value}} --{{opt2}} {{value}}`
```

Comment:
I think we can remove this line. The following lines are concrete examples of this. And we usually prefer to keep concrete examples rather than generic ones like these.